Demo/src/main/java/com/braintreepayments/demo/ShopperInsightsFragment.kt

```diff
@@ -139,7 +139,7 @@ class ShopperInsightsFragment : BaseFragment(), PayPalListener, VenmoListener {
         shopperInsightsClient.sendPayPalSelectedEvent()
         payPalClient.tokenizePayPalAccount(
             requireActivity(),
-            PayPalRequestFactory.createPayPalVaultRequest(activity, emailInput.editText?.text.toString())
+            PayPalRequestFactory.createPayPalVaultRequest(activity, emailInput.editText?.text.toString(), "")
```

Comment:
should we pass the phone input edit text value here instead of ""?

Reply:
This is in the ShopperInsightsFragment in the demo. Would we want to have an edit text on that screen as well as the PayPalFragment?

Reply:
Oh no sorry missed that this was in shopper insights!